sdk.properties

```diff
@@ -1 +1 @@
-excludeModels=verifiedpermissions.2021-12-01
+# onlyIncludeModels=verifiedpermissions.2021-12-01
```

Comment:
Revert this prior to merge.  Changed to allow for retesting of the Model Integration CI task with a failure condition.

Reply:
Done